content/en/platform/corda/5.1/deploying-operating/observability/metrics/task-manager.md

```diff
@@ -0,0 +1,57 @@
+| `corda_taskmanager_executor_idle_seconds_count`           | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_idle_seconds_max`             | Gauge   | `name`                                                    | The maximum observed value for the base metric.                                                                                       |
+| `corda_taskmanager_executor_idle_seconds_sum`             | Counter | `name`                                                    | The total sum of all observed values for the base metric.                           |
+| `corda_taskmanager_executor_pool_core_threads`            | Gauge   | `name`                                                    | The core number of threads for the pool.                                               |
+| `corda_taskmanager_executor_pool_max_threads`             | Gauge   | `name`                                                    | The maximum allowed number of threads in the pool.                                     |
+| `corda_taskmanager_executor_pool_size_threads`            | Gauge   | `name`                                                    | The current number of threads in the pool.                                             |
+| `corda_taskmanager_executor_queue_remaining_tasks`        | Gauge   | `name`                                                    | The number of additional elements that this queue can ideally accept without blocking. |
+| `corda_taskmanager_executor_queued_tasks`                 | Gauge   | `name`                                                    | The approximate number of tasks that are queued for execution.                         |
+| `corda_taskmanager_executor_scheduled_repetitively_total` | Counter | `name`                                                    |                                                                                        |
+| `corda_taskmanager_executor_seconds_count`                | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_seconds_max`                  | Gauge   | `name`                                                    | The maximum observed value for the base metric.                                                                                       |
+| `corda_taskmanager_executor_seconds_sum`                  | Counter | `name`                                                    | The total sum of all observed values for the base metric                            |
+| `corda_taskmanager_completion_time_count`                  |  Counter  | <ul><li>`task.manager.name`</li><li>`task.type`</li></ul> | The number of tasks completed by a task manager..            |
```

Comment:
Having it go from no bullet points to bullet points looks a bit odd. Would separating them with a comma be an option? Also, double full stop at end of this line.